formula_renames.json

```
@@ -152,6 +152,7 @@
   "sonar": "sonarqube",
   "speedtest_cli": "speedtest-cli",
   "ssreflect": "math-comp",
+  "synergy-core":"deskflow",
```

Would it also make sense to do this?

```suggestion
  "synergy":"deskflow",
  "synergy-core":"deskflow",
```